src/main/kotlin/app/revanced/patches/all/misc/debugging/patch/EnableAndroidDebuggingPatch.kt

```diff
@@ -9,7 +9,7 @@ import org.w3c.dom.Element
 
 @Patch(false)
 @Name("Enable android debugging")
-@Description("Enables Android debugging capabilities.")
+@Description("Enables Android debugging capabilities. This patch is only for software development and it may cause app slow down.")
```

Comment:
Due to correctness I'd suggest:

```suggestion
@Description("Enables Android debugging capabilities. This can slow down the app.")
```